### PR TITLE
feat(installer): Introduce flags to enable / disable Keptn services

### LINKED ISSUE #8075: Enable / Disable all services by Helm chart
Ability to enable / disable any services in the control plane chart.

For example what if I don't want the statistics-service just for a temporary demo? I shouldn't have to use kustomise for this.

@mowies ref our discussion on #6042

### PR DESCRIPTION
**Pull request by warber**
closes #8075

### CODE REVIEW

File: installer/manifests/keptn/templates/lighthouse-service.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+{{- if .Values.approvalService.enabled }}
```
Comment (mowies) [inline installer/manifests/keptn/templates/lighthouse-service.yaml:1]:
```suggestion
{{- if .Values.lighthouseService.enabled }}
```

File: installer/manifests/keptn/templates/remediation-service.yaml [hunk 2]
```diff
@@ -77,4 +77,4 @@ spec:
       protocol: TCP
   selector: {{- include "keptn.common.labels.selectorLabels" . | nindent 4 }}
     app.kubernetes.io/name: remediation-service
-
+{{- end}}
```
Comment (mowies) [inline installer/manifests/keptn/templates/remediation-service.yaml:80]:
```suggestion
{{- end }}
```

File: installer/manifests/keptn/templates/rbac.yaml [hunk 3]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (mowies) [inline installer/manifests/keptn/templates/rbac.yaml:1]: Please also make sure that the corresponding lighthouse-svc service account and rolebinding is not created when the service is disabled.

**Review by thisthat — COMMENTED** [review]
Maybe we can also skip the creation of the service accounts if the svc is not enabled

**Review by mowies — APPROVED** [review]
LGTM!

**Comment by warber** [comment]
@thisthat  done as part of the review commits :+1:

**Pull request merged**